photon-client/src/main/scala/com/linkedin/photon/ml/estimators/GameEstimator.scala

```diff
@@ -208,7 +195,7 @@ class GameEstimator(val params: GameParams, val sparkContext: SparkContext, val
 
     val labelAndOffsetAndWeights = gameDataSet.mapValues(gameData =>
       (gameData.response, gameData.offset, gameData.weight))
-      .setName("Training label and offset and weights")
+      .setName("Training labels, offsets and weights")
```

Comment:
Sure - I agree. I was talking about the trailing comma before the final item in a list (before the 'and'):
```
Training labels, offsets, and weights.
```